src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/AbstractCloudFoundryDeployer.java

```diff
@@ -65,4 +81,10 @@ String buildpack(AppDeploymentRequest request) {
 			.orElse(this.deploymentProperties.getBuildpack());
 	}
 
+	Predicate<Throwable> isNotFoundError() {
+		return t -> {
+			logger.debug("isNotFoundError? Got Exception" + t);
```

Comment:
This can be removed, was for testing purposes only but forgot to remove. Will do